pkg/agent/openflow/multicast.go

```diff
@@ -91,6 +97,13 @@ func (f *featureMulticast) replayFlows() []*openflow15.FlowMod {
 	return getCachedFlowMessages(f.cachedFlows)
 }
 
+func (f *featureMulticast) multicastFlexibleIPAMGroup(groupID binding.GroupIDType) binding.Group {
+	group := f.bridge.NewGroupTypeAll(groupID)
```

Comment:
Is it possible to use Flow actions like "output:uplink,hostOFPort" to replace group?

Reply:
+1 to the question, and if not, then is it possible to leverage `InstallMulticastGroup` to install the group, instead of having dedicated functions for each mode.

Reply:
Updated. Thanks. Tested the flow action `output:uplink,hostOFPort`, and it works.